src/homepageExperience/components/steps/java/WriteDataSql.tsx

```diff
@@ -0,0 +1,264 @@
+import React, {useContext, useEffect, useState} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
+import {
+  Columns,
+  ComponentSize,
+  Grid,
+  InfluxColors,
+  Panel,
+  Table,
+} from '@influxdata/clockface'
+
+import {SafeBlankLink} from 'src/utils/SafeBlankLink'
+import WriteDataHelperBuckets from 'src/writeData/components/WriteDataHelperBuckets'
+import CodeSnippet from 'src/shared/components/CodeSnippet'
+import {WriteDataDetailsContext} from 'src/writeData/components/WriteDataDetailsContext'
+
+import {getOrg} from 'src/organizations/selectors'
+import DataListening from 'src/homepageExperience/components/DataListening'
+import {getBuckets} from 'src/buckets/actions/thunks'
+import {event} from 'src/cloud/utils/reporting'
+
+const logCopyCodeSnippet = () => {
```

Comment:
Scope these functions within the component.

Reply:
Fixed